### ISSUE TITLE
Use XYUV axes lables

### ISSUE THREAD

**Issue by monkeypants** [monkeypants/foamcutter#3]
Manual says (re [TRAJ]):

```
COORDINATES = X Y Z - The names of the axes being controlled. Only
X, Y, Z, A, B, C, U, V, W are valid. Only axes named in COORDINATES 
are accepted in g-code. This has no effect on the mapping from G-code 
axis names (X- Y- Z-) to joint numbers—for trivial kinematics, X is always
joint 0, A is always joint 3, and U is always joint 6, and so on. It is permitted
to write an axis name twice (e.g., X Y Y Z for a gantry machine) but this 
has no effect.
```

So, I guess that means I need to use 8 axis.

**Comment by monkeypants** [comment]
Note also configuration of foamcutter simulation in the sim.axis:
- AXES = 9
- COORDINATES = X Y U V
- AXES_{0,1,2,6,7,8} are all TYPE = LINEAR, what's with that?

**Comment by monkeypants** [comment]
fixed by 17a5926
